PowerShellTools/ServiceManagement/PowershellHostProcessException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PowerShellTools.ServiceManagement
+{
+    [Serializable]
+    public class PowershellHostProcessException : Exception

```

Comment:
Why do we need this since we should be able to serialize Exception directly? Or you are thinking about to do specific action for its handling?